app/controllers/ukrentaroom/RentalsAndRaRCheckYourAnswersController.scala

```diff
@@ -0,0 +1,131 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.ukrentaroom
+
+import audit.RentARoomAuditModel._
+import audit.RentalsAndRentARoomAuditModel._
+import audit.{AuditService, RentARoomAuditModel, RentalsAndRentARoomAuditModel}
+import com.google.inject.Inject
+import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import models.requests.DataRequest
+import models.{JourneyContext, PropertyType, RaRAbout, RentARoom, RentalsAndRaRAbout, RentalsAndRentARoom}
+import play.api.Logging
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.libs.json.Writes
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import service.PropertySubmissionService
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import viewmodels.checkAnswers.adjustments.BalancingChargeSummary
+import viewmodels.checkAnswers.ukrentaroom.{ClaimExpensesOrRRRSummary, TotalIncomeAmountSummary, UkRentARoomJointlyLetSummary}
+import viewmodels.govuk.summarylist._
+import views.html.ukrentaroom.{CheckYourAnswersView, RentalsAndRaRCheckYourAnswersView}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RentalsAndRaRCheckYourAnswersController @Inject() (
+  override val messagesApi: MessagesApi,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  propertySubmissionService: PropertySubmissionService,
+  view: RentalsAndRaRCheckYourAnswersView,
+  audit: AuditService
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController with I18nSupport with Logging {
+
+  def onPageLoad(taxYear: Int): Action[AnyContent] =
+    (identify andThen getData andThen requireData) { implicit request =>
+      val ukRentARoomJointlyLetSummary =
+        UkRentARoomJointlyLetSummary.row(
+          taxYear,
+          request.userAnswers,
+          request.user.isAgentMessageKey,
+          RentalsAndRentARoom
+        )
+      val totalIncomeAmountSummary =
+        TotalIncomeAmountSummary.row(taxYear, request.userAnswers, request.user.isAgentMessageKey, RentalsAndRentARoom)
+      val claimExpensesOrRRRSummary =
+        ClaimExpensesOrRRRSummary.rows(taxYear, request.user.isAgentMessageKey, request.userAnswers) //Todo: Update in related ticket to parameterise
+
+      val balancingChargeSummary =
+        BalancingChargeSummary.row(taxYear, request.userAnswers) //Todo: Update in related ticket to parameterise
```

Comment:
BalancingCharge in about section?

Reply:
Hi Arun, from the prototype, there is balancing charge as far as I can see